azurelinuxagent/common/utils/networkutil.py

```diff
@@ -179,34 +167,30 @@ def get_accept_tcp_rule(command, destination, firewalld_command="", wait=""):
             cmd = AddFirewallRules.__get_firewalld_base_command(firewalld_command)
         else:
             cmd = AddFirewallRules.__get_iptables_base_command(wait)
-        cmd = cmd + ['-t', 'security', command, 'OUTPUT', '-d', destination, '-p', 'tcp', '--destination-port', '53', '-j', 'ACCEPT']
+        cmd = cmd + ['-t', 'security', command, 'OUTPUT', '-d', destination, '-p', 'tcp', '--destination-port', '53',
+                     '-j', 'ACCEPT']
         return cmd
 
     @staticmethod
-    def get_iptables_accept_command(wait, command, destination, owner_uid):
-        cmd = AddFirewallRules.__get_iptables_base_command()
-        cmd.extend(AddFirewallRules.__get_common_accept_command_params(wait, command, destination, owner_uid))
-        return cmd
+    def get_accept_command(command, destination, owner_uid, firewalld_command="", wait=""):
```

Comment:
NIT: `get_accept_command` does not make complete sense here, maybe rename it to `get_wire_root_accept_rule`?
```suggestion
    def get_wire_root_accept_rule(command, destination, owner_uid, firewalld_command="", wait=""):
```